server/gabble/chat/socket.go

```diff
@@ -40,24 +36,25 @@ func MakeServer(host string, port string) (*SocketIOServer, error) {
 
 	server := new(SocketIOServer)
 
-	server.port = port
-	server.socket = socket
-	server.host = host
-	server.setEventListeners()
+	server.port = conf.GetPort()
+	server.host = conf.GetHost()
+	server.Socket = socket
```

Comment:
Turn `socket` property *public* so it can be used by the `InitEvents`-like functions